src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -95,14 +97,16 @@ export const TemplatesCatalogDrawerCreateForm: FC<TemplatesCatalogDrawerCreateFo
 
       const nameParameterExists = isNameParameterExists(template);
 
-      const templateToProcess = produce(template, (draftTemplate) => {
+      const templateToProcess = produce(template, (draftTemplate: V1Template) => {
```

Comment:
If you are adding types, what about template and function?

Reply:
"More" doesn't always mean "better".